docs/Authenticating-Users-via-SAML.md

```diff
@@ -18,8 +18,6 @@ In its simplest terms, SAML boils down to four terms:
 
 * **authentication**:  a means of verifying that a user is who they purport to be.  Authentication is performed by the identify provider, by extracting the user name and password provided in a login form, and matching this with information stored in a database.
 
-* **authorization**:  defines resources a user can access.  When authorization is turned on with cBioPortal, users can only access cancer studies they are specifically authorized to view (or PUBLIC studies, [if the server has been configured to contain these](User-Authorization.md#configuring-public-studies)).  This enables one to store multiple cancer studies within a single instance of cBioPortal, but provide fine-grained control over which users can access which studies.  Authorization is implemented within the core cBioPortal code, and not the identify provider.
```

Comment:
similar comment as above. Here they mention "public studies" which is no longer a functional feature if I understood @n1zea144 correctly .. so it is good we are removing this reference.